examples/OpenGL45/arrayReductionByMax_version3.shadertrap

```diff
@@ -0,0 +1,70 @@
+
+CREATE_BUFFER input_array_expected SIZE_BYTES 1024 INIT_VALUES uint 256 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127 128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223 224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239 240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255
+CREATE_BUFFER result SIZE_BYTES 4 INIT_VALUES uint 0
+CREATE_BUFFER expected SIZE_BYTES 4 INIT_VALUES uint 256
+
+BIND_SHADER_STORAGE_BUFFER BUFFER input_array BINDING 0
+BIND_SHADER_STORAGE_BUFFER BUFFER result BINDING 1
+
+DECLARE_SHADER shader KIND COMPUTE
+#version 450
+
+layout(local_size_x=256, local_size_y=1, local_size_z=1) in;
+
+layout(std430, binding = 0) buffer input_array {
+  uint[256] data;
+}; 
+layout(std430, binding = 1) buffer result {
+  uint res;
+};
+
+void main() {
+  // id range: [0, 255].
+  uint id = uint(gl_LocalInvocationID.x);
+  // depth of the tree where we perform reduction. 
+  uint Depth = findMSB(data.length());
+  
+  // At each step, half of invocations drop out of the computation. 
+  // Index corresponding to active invocations will update with max value while
+  // the rest will set to corresponding id in case race condition between multiple threads occurs.
+  // This way each position in the input array other than index 0 should be set to its index value
+  // by the corresponding invocation.  
+  // Each active invocation will add its value to that of a neighboring invocatiion. 
```

Comment:
Please fix all examples so that they refer to the correct operation instead of "add".

Reply:
Done.